e2e-tests/desktop/fixtures/tesseractTest.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { test } from '@playwright/test';
+import { createWorker } from 'tesseract.js';
+
+export const textToMatch =
+  /To run a command as administrator|Welcome to OpenSSH Server/;
+
+export const tesseractTest = test.extend({
+  tesseract: [
+    async ({}, use) => {
+      const worker = await createWorker('eng', 1, {
+        cachePath: './artifacts',
```

Comment:
question for learning purposes: On test failures what type of data is being stored in the artifacts folder? Based on the the [docs](https://github.com/naptha/tesseract.js/blob/master/docs/api.md#:~:text=common%20to%20modify-,cachePath,-path%20for%20the) it says `traineddata` but not exactly sure what that means in this context.

Reply:
So for this `tesseract` package, nothing is generated from test failures that gets stored in `artifiacts`. `traineddata` is actually just the language files that gets downloaded when running for the first time (which is why it can be slow) that it uses for its engine to actually run. It's basically where we store the setup files that are needed for this package.